src/RemoteStorageService/Check.js

```diff
@@ -0,0 +1,24 @@
+import { useRemoteStorageMappings } from '@folio/stripes/smart-components';
+import { useHoldings } from '../providers';
+
+
+export const useByLocation = () => {
+  const remoteMap = useRemoteStorageMappings();
+
+  return ({ fromLocationId, toLocationId }) => (fromLocationId in remoteMap) && !(toLocationId in remoteMap);
+};
+
+
+export const useByHoldings = () => {
+  const { holdingsById } = useHoldings();
+  const check = useByLocation();
+
+  return ({ fromHoldingsId, toHoldingsId }) => {
```

Comment:
@axelhunn I wonder if this should be wrapped  in `useCallback`. Otherwise I think it will be recreated on every render. What do you think?

Reply:
Since I've read this [article by Kent C. Dodds](https://kentcdodds.com/blog/usememo-and-usecallback), I try not to overuse the `useCallback`.

But you're right, this function is used in dependencies of another `useCallback`s and `useMemo`s.
Fixed it.

Reply:
Thank you for sharing that article @axelhunn